kolibri/deployment/default/settings/base.py

```diff
@@ -19,6 +19,10 @@
 import pytz
 from django.conf import locale
 from tzlocal import get_localzone
+try:
+    from urlparse import urljoin
+except ImportError:
+    from urllib.parse import urljoin
```

Comment:
Move to `kolibri.utils.compat`

Reply:
Seems like we should actually just be using: `from six.moves.urllib.parse import urljoin`